src/harness/rwcRunner.ts

```diff
@@ -79,7 +79,7 @@ namespace RWC {
                         tsconfigFiles.push({ unitName: tsconfigFile.path, content: tsconfigFileContents.content });
                         const parsedTsconfigFileContents = ts.parseJsonText(tsconfigFile.path, tsconfigFileContents.content);
                         const configParseHost: ts.ParseConfigHost = {
-                            useCaseSensitiveFileNames: Harness.IO.useCaseSensitiveFileNames(),

```

Comment:
It seems like this is something we need to capture and read from the rwc capture file.

Reply:
And now it does. No captures should need to be updated, since `undefined` is coercible to false, and they all execute correctly under case insensitivity as it currently stands.